src/shippingservice/src/shipping_service/quote.rs

```diff
@@ -0,0 +1,32 @@
+use core::fmt;
+
+use opentelemetry::{trace::get_active_span, KeyValue};
+
+#[derive(Debug, Default)]
+pub struct Quote {
+    pub dollars: i64,
+    pub cents: i32,
+}
+
+// TODO: Check product catalog for price on each item (will likley need item ID)
+pub fn create_quote_from_count(count: u32) -> Quote {
+    let f = if count == 0 { 0.0 } else { 8.99*(count as f64) };
+    get_active_span(|span| {
+        let q = create_quote_from_float(f);
+        span.add_event("Quote Issued", vec![KeyValue::new(format!("{}", q), true)]);
```

Comment:
I've re-built the image and ran the demo, but this event never showed up in Jaeger.

Reply:
i'll look into it

Reply:
I think I tracked something down. I noticed that `get_active_span` does... expect the span to be active :)

I specifically marked the span as active on local -- building and will update if it works!

Reply:
I think that fixed it? Let me push and you see for yourself :) 
![image](https://user-images.githubusercontent.com/7660110/178306585-771be715-5bb2-48cf-bdb8-4ffff85c7c2a.png)